src/components/EmployeeDashboard/lists/BoroughTopIssueList.jsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import IssueListEntry from './IssueListEntry.js';
+import IssueListEntry from './IssueListEntry.jsx';
```

Comment:
Imports should not contain the extension unless there is more than one file with the same name.